box/box.py

```diff
@@ -436,12 +444,13 @@ def __setstate__(self, state):
 
     def __get_default(self, item, attr=False):
         default_value = self._box_config["default_box_attr"]
+        config = self.__box_config(extra_namespace=item)
```

Comment:
I know it's super handy to do this at the start of the function here. But for now will need to keep it in each `if` block, as this would slow down processes that don't use one of the defaults that require it.

This code wasn't designed for speed from the start, but enough people use it in "semi performant" applications I try to think a little bit about not slowing down some paths they might have optimized for at least 🤷